pkg/server/ca/manager.go

```diff
@@ -974,6 +971,19 @@ func parseUpstreamCACSRResponse(resp *upstreamca.SubmitCSRResponse) ([]*x509.Cer
 	return certChain, trustBundle, nil
 }
 
+func parseCertificates(certChain [][]byte) ([]*x509.Certificate, error) {
```

Comment:
nit: `certChain` name implies these certificates are related, which isn't always the case (i.e. roots) nor is it important to the functionality of the method. Maybe `rawCerts` instead of `certChain` and `rawCert` instead of `chain`? or something like that?

Reply:
done